tidb-cloud/select-cluster-tier.md

```diff
@@ -29,6 +29,22 @@ For each TiDB Cloud account, you can use one complimentary Developer Tier cluste
 
 The one-year free trial begins the day the first Developer Tier cluster is created.
 
+### User name prefix
+
+<!--Important: Do not update the section name "User name prefix" because this section is referenced by the TiDB error messages.-->
+
+For each Developer Tier TiDB cluster, TiDB Cloud generates a unique prefix to distinguish it from others.
+
+When you connect to a Developer Tier TiDB cluster via standard connection or when you create a database user, you must include the prefix in the user name and wrap the name with quotation marks.
+
+Take the default `root` user an example. If the prefix for your cluster is `3pTAoNNegb47Uc8`, you need to use `'3pTAoNNegb47Uc8.root'` to connect to your TiDB cluster.
```

Comment:
For example, if the prefix for your cluster is `3pTAoNNegb47Uc8` and you must create a user with such a prefix, (eg:`3pTAoNNegb47Uc8.test`) and connect to your database with the username which has the prefix.

For example, your must connect to your cluster with username `3pTAoNNegb47Uc8.root`:

```shell
mysql --connect-timeout 15 -u '3pTAoNNegb47Uc8.root' -h gateway01.us-east-1.staging.shared.aws.tidbcloud.com -P 4000 -D test -p
```

When you create a new database user, you must create a user which has the same prefix:
```mysql
CREATE USER `3pTAoNNegb47Uc8. jeffrey`;
```